application/src/main/java/com/softserve/lv460/application/security/jwt/JwtTokenProvider.java

```diff
@@ -0,0 +1,105 @@
+package com.softserve.lv460.application.security.jwt;
+
+import io.jsonwebtoken.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+
+@Component
+public class JwtTokenProvider {
+
+  private static final Logger logger = LoggerFactory.getLogger(JwtTokenProvider.class);
+
+  @Value("${jwtconf.SECRET}")
+  private String jwtSecret;
+
+  @Value("${jwtconf.EXPIRATIONTIME}")
```

Comment:
you're injecting the same property in multiple classes. You should have just single class with security-related properties and inject it where needed.